dev-env/lib/dade-dump-profile

```diff
@@ -128,3 +128,8 @@ fi
 echo "export LC_ALL=en_US.UTF-8"
 echo "export LANG=en_US.UTF-8"
 
+echo "# Used when running Bazel commands with --config=oracle"
+echo "export ORACLE_USERNAME=system"
+echo "export ORACLE_PWD=hunter2"
+echo "export ORACLE_PORT=1521"
```

Comment:
I'm _definitely_ not sure of whether this is the correct place to define it. My main intent is to share these variables across CI and a normal development setup without having to define these separately (as it was done so far in `ci/build.yml`). If you feel that these variables belong somewhere else I'm happy to move it, but I thing it would be beneficial to leave these in a place where they are shared across CI and a normal development setup without ad-hoc ways of defining them.

Reply:
no idea tbh but this doesn’t seem that wrong